packages/core/core/src/types.ts

```diff
@@ -15,7 +15,7 @@ export interface UseProduct<PRODUCT, PRODUCT_FILTERS, SORTING_OPTIONS> {
     term?: any;
     filters?: PRODUCT_FILTERS;
     [x: string]: any;
-  }) => Promise<void>;
+  }, customQuery: CUSTOM_QUERY) => Promise<void>;
```

Comment:
this will display `customQuery` for each integration while GQL is not a part of each

Reply:
Done.